src/docs/cmdstan-guide/running.tex

```diff
@@ -223,212 +223,37 @@ \subsubsection{Windows}
 
 \section{Command-Line Options}\label{stan-command-line-options.section}
 
-\CmdStan executables are highly configurable, allowing the user to specify
-and customize not only the calculation method but also the data, output,
-initialization, and random number generation.  The arguments are defined
-hierarchically so that, for example, optimization settings are not necessary
-when sampling.
-
-The atomic elements of the hierarchy (i.e., those without
-corresponding values) are \textit{categorical arguments} (sometimes
-called ``flags'') which define self-contained categories of arguments.
-
-\CmdStan's commands have more hierarchical structure than is typical of
-command line executables, which usually have at most two subgroups of
-commands.  Arguments grouped within a category are not ordered with
-respect to each other.  The only ordering is that the global options
-come before the method argument and subcommand-specific options after
-the method argument.  For example, the following four commands all
-define the same configuration:%
-%
-\footnote{
-The backslash (\code{\textbackslash}) is used at the end of a line in
-a command to indicate that it continues on the next line.  The
-indentation to indicate the structure of the command is for
-pedagogical purposes only; the same result would be obtained writing
-each command on one line with single spaces separating the elements.
-}
+A \CmdStan program can be run in many different ways, e.g., sampling,
```

Comment:
[optional]
We can get rid of the `\` in `\CmdStan`.  It was overconfigured when I first wrote it.

Reply:
done